src/main/java/edu/harvard/iq/dataverse/search/savedsearch/SavedSearchServiceBean.java

```diff
@@ -240,6 +253,37 @@ public JsonObjectBuilder makeLinksForSingleSavedSearch(DataverseRequest dvReq, S
         return response;
     }
 
+    public void unLinksForSingleSavedSearch(DataverseRequest dvReq, SavedSearch savedSearch) throws SearchException, CommandException {
+        logger.info("UNLINK SAVED SEARCH (" + savedSearch.getId() + ") START search and unlink process");
+        Date start = new Date();
+        Dataverse linkingDataverse = savedSearch.getDefinitionPoint();
+
+        SolrQueryResponse queryResponse = findHits(savedSearch);
+        for (SolrSearchResult solrSearchResult : queryResponse.getSolrSearchResults()) {
+
+            DvObject dvObjectThatDefinitionPointWillLinkTo = dvObjectService.findDvObject(solrSearchResult.getEntityId());
+            if (dvObjectThatDefinitionPointWillLinkTo == null) {
+                continue;
+            }
+
+            if (dvObjectThatDefinitionPointWillLinkTo.isInstanceofDataverse()) {
+                Dataverse linkedDataverse = (Dataverse) dvObjectThatDefinitionPointWillLinkTo;
+                DataverseLinkingDataverse dvld = dvLinkingService.findDataverseLinkingDataverse(linkedDataverse.getId(), linkingDataverse.getId());
+                if(dvld != null) {
+                    Dataverse dv = commandEngine.submitInNewTransaction(new DeleteDataverseLinkingDataverseCommand(dvReq, linkingDataverse, dvld, true));
```

Comment:
Seems not enough to reindex unlinked Dataverse. But that's another topic, database is cleared.

Reply:
Sorry, I'm not sure what you mean. Are you concerned about something?

Reply:
I see. Thanks for clarifying.